content/news/ragls6-registrations.md

```diff
@@ -0,0 +1,13 @@
+<div style="text-align:center" markdown="1">
+![Red Alert Global League Season 6](/images/news/20181003-ragl-vi.png)
+</div>
+
+Attention, players!
+
+Red Alert Global League Season 6 is starting soon, with registrations closing on October 10th. The tournament itself is to be held from October 15th till December 23rd, ending with a Masters Finals to decide the next League champion.
+
+New to this season is the streamlining of the tier system, with Masters and Minions divisions set to be potentially increased (up to 16 players per group) and Recruits division removed in favour of pre-season qualification stage that will be used to rank league newcomers and fill vacant Minions division slots. Also, the main stage is getting packed with action now, as league rounds are double-stacked per week, and each player is to play two best-of-2 matches in his division each week.
+
+Top 4 Masters and Minions Champion are set to receive the major part of the prize pool as usual, but now every participant who finishes the season proper is selected for a random draw of 3 minor prizes, so the consistency and dedication to the tournament will definitely pay off.
```

Comment:
`Top 4 Masters and Minions Champion` &rarr; `The top 4 Masters and Minions Champions`?  I would also break this into two sentences `of three minor prizes.  Consistency and dedication to the tournament will definitely pay off!`